fbink.c

```diff
@@ -1798,6 +1798,29 @@ void
 	    penBGColor);
 }
 
+// Dump a few of out internal state variables to struct pointed to by fbink_state
+void
+	fbink_get_state(const FBInkConfig* fbink_config, FBInkState* fbink_state)
+{
+	if (fbink_state) {
+		fbink_state->view_width     = viewWidth;
+		fbink_state->view_height    = viewHeight;
+		fbink_state->bpp            = vInfo.bits_per_pixel;
+		fbink_state->font_w         = FONTW;
+		fbink_state->font_h         = FONTH;
+		fbink_state->font_sz_mult   = FONTSIZE_MULT;
+		fbink_state->font_name      = fontname_to_string(fbink_config->fontname);
```

Comment:
... wait, there may be a bit of C shenanigan involved where it just returns a pointer to the ro data in this specific instance, in which case, everything's good.

5AM brain, plus, err, C :D.

Will have to double-check, I'm not quite sure that applies here, since it's not a static const or a define, just a string litteral in a return statement...